### PR TITLE
docs: add page in the docs describing argilla user roles

### LINKED ISSUE #3806: [DOCS] Add page in the docs describing Argilla user roles
## Which page or section is this issue related to?

Right now Argilla has three roles for its users: owner, admin and annotator. We've briefly mention about these roles in several pages of the documentation, but it would be good to have an specific page describing the roles (basically what each role can do and what cannot do)

This image can be used as a reference (thanks @dvsrepo!):
Argilla roles table

### PR DESCRIPTION
**Pull request by sdiazlor**
# Description

We updated the user management page:
- more info about the user roles
- adding the photograph as a visual schema
- fixing a non-working link
- fixing some typos

Closes #3806 

**Type of change**

(Remember to title the PR according to the type of change)

- [X] Documentation update

**How Has This Been Tested**

(Please describe the tests that you ran to verify your changes.)

- [X] `sphinx-autobuild` (read [Developer Documentation](https://docs.argilla.io/en/latest/community/developer_docs.html#building-the-documentation) for more details)

**Checklist**

- [X] I added relevant documentation
- [X] I followed the style guidelines of this project
- [X] I did a self-review of my code
- [X] I made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I filled out [the contributor form](https://tally.so/r/n9XrxK) (see text above)
- [ ] I have added relevant notes to the `CHANGELOG.md` file (See https://keepachangelog.com/)

### CODE REVIEW

**Comment by davidberenstein1957** [comment]
@sdiazlor normally when working on the docs, you should also create a branch from main, in order to merge it to main lateron :) no problem for now but good it keep in mind tomorrow.

File: docs/_source/getting_started/installation/configurations/user_management.md [hunk 1]
```diff
@@ -1,26 +1,49 @@
...
+- **Feedback**: It can provide feedback with the Argilla UI.
+
+## Annotator
+
+An annotator user is limited to accessing only the datasets assigned to it within the workspace. It has two specific permissions:
+
+- **Annotation**: It can annotate datasets in the Argilla UI.
+- **Feedback**: It can provide feedback with the Argilla UI.
+
+![user_roles_schema](../../../_static/images/installation/user_management/user_roles.png)
```
Comment (davidberenstein1957) [inline docs/_source/getting_started/installation/configurations/user_management.md]: I belive you can use a direct relative reference here.
`(/_static/images/installation/user_management/user_roles.png)`

**Comment by sdiazlor** [comment]
> @sdiazlor normally when working on the docs, you should also create a branch from main, in order to merge it to main lateron :) no problem for now but good it keep in mind tomorrow.

Okay, yeah I had that in mind, but I realized now that I used the forking made for the tutorial and by then I had chosen to fork the develop branch. I'll take care

**Comment by sdiazlor** [comment]
@davidberenstein1957 The relative path is changed.

**Review by davidberenstein1957 — APPROVED** [review]

**Pull request merged**